foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/internal/oxm/XMLConversionManager.java

```diff
@@ -1641,12 +1641,7 @@ protected Byte[] convertSchemaBase64ToByteObjectArray(Object sourceObject) throw
 
     @Override
     public String buildBase64StringFromBytes(byte[] bytes) {
-        byte[] convertedBytes = Base64.base64Encode(bytes);
-        StringBuilder buffer = new StringBuilder();
-        for (int i = 0; i < convertedBytes.length; i++) {
-            buffer.append((char) convertedBytes[i]);
-        }
-        return buffer.toString();
+        return java.util.Base64.getEncoder().encodeToString(bytes);
```

Comment:
I'd prefer see new method in `org.eclipse.persistence.internal.oxm.conversion.Base64` which is called from here over spreading usage of `java.util.Base64` through the codebase leading to non-explicit imports (ie when one sees Base64 in the code, can it be determined which package it comes from without looking at import statements?)

Reply:
I am just checking that the JDK implementation is also creating a new byte array anyway, so this fix will not help.